Recipes/Apply_function_to_every_gridpoint.ipynb

```diff
@@ -37,12 +37,13 @@
   },
```

Comment:
<p>@adele-morrison / @navidcy et others - Do we have consensus on fixing the projection for the triple on these global plots (e.g. <a href="https://github.com/COSIMA/cosima-recipes/issues/432" target="_blank">https://github.com/COSIMA/cosima-recipes/issues/432</a>) ? Should we put a comment in everytime sayings its odd north of 65N? Or load the grid everytime and plot "correctly" ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>From my "I have no idea what this plot scientifically means, but having examples of various technical tricks is handy" viewpoint, having the example of loading the correct grid would be useful.</p>

Reply:
This is one example:

https://cosima-recipes.readthedocs.io/en/latest/Tutorials/Making_Maps_with_Cartopy.html#Fixing-the-tripole

Reply:
<p>I suggest adding a note above this cell.</p><p> </p><p>"For simplicity, we haven't corrected for the tripolar projection (link) used in the ACCESS ocean models. Please note that north of 65°N, this figure is distorted."</p>

Reply:
let's just fix the plot, why not?

Reply:
I've done it. Now it's there correct for future generations to see.